src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java

```diff
@@ -299,6 +303,7 @@ public void should_not_delete_blob_unless_processed()
         CloudBlockBlob blob = testContainer.getBlockBlobReference(VALID_ZIP_FILE_WITH_CASE_NUMBER);
         await("file should not be deleted")
             .atMost(5, SECONDS)
+            .pollInterval(1, TimeUnit.SECONDS)
```

Comment:
same, here. Check file still exists after a while

Reply:
Same as above. It waits the full timeout unless there is no need for it.